src/stack.py

```diff
@@ -0,0 +1,23 @@
+"""This module defines the Stack data structure."""
+
+from linked_list import Linked_List
+
+
+class Stack(object):
+    """The stack data structure is a composition of the Linked List structure."""
+    def __init__(self, iterable=None):
+        """Initialize stack as a Linked_List-esque object."""
```

Comment:
There's no need to reference the Linked List that composes this data structure in the doc strings. It's a stack. Let it just be a stack.